### ISSUE TITLE
NoDataError: empty CSV

### ISSUE THREAD

**Issue by peipp410** [MGI-tech-bioinformatics/DNBelab_C_Series_HT_scRNA-analysis-software#93]
Hello! I encountered the following error. I tried to check the source code but I can't locate where the function is because there is no file called 'oligo_filter.py'. What's the possible cause of this error and how to fix that? Thanks!
![WechatIMG2977](https://github.com/user-attachments/assets/1517cdee-494a-4064-a5f2-948c72fa644f)

**Comment by lishuangshuang0616** [comment]
Can you check the results in **01.data/oligo.sequencing.report.csv** and also provide the screenshot of the results from **01.data/oligo.reads.fq.gz**?

**Comment by peipp410** [comment]
![29901723521993_ pic_hd](https://github.com/user-attachments/assets/3123cf6e-d588-48f7-95e6-33a5cc967ddc)
None of the reads passed QC, which is so wired. The fastq file is also empty. How could that happen?

**Comment by peipp410** [comment]
p.s.  with arguments "--chemistry scRNAv2HT --darkreaction unset,unset"

**Comment by lishuangshuang0616** [comment]
Screenshot the first 10 sequences R1/R2 of cDNA and oligo, I'll take a look

**Comment by peipp410** [comment]
![552a16b5eb225d30ccb124e835663225](https://github.com/user-attachments/assets/0e47ad9b-72e7-41c1-b113-e6463c48b270)
Thanks! Problem solved. 'N'-base exists.

**Comment by lishuangshuang0616** [comment]
Yes, this version will cause reads to be filtered for oligo fastq containing N. Currently, you can only manually remove the fixed sequence. 
This problem will be solved in the upcoming 2.1.3 version.
